shell/app-shell/elements/background-arcs/bg-arc.js

```diff
@@ -56,7 +56,8 @@ class BgArc extends Xen.Debug(Xen.Base, log) {
   _render(props, state) {
     return [props, state];
   }
-  async _updateKey(key) {
+  async _updateKey(userid, key) {
+    Firebase.db.child(`users/${userid}/arcs/${key}/touched`).set(Firebase.firebase.database.ServerValue.TIMESTAMP);
```

Comment:
FYI @lindner this will be another place to update for genericizing backing persistent data store.

Reply:
This is hacked in for demo timeline. My intent is to store user's arc metadata in an arc.